clm/mlperf_logging_utils.py

```diff
@@ -153,6 +163,22 @@ def on_step_end(
         Event called at the end of a training step.
         """
         if state.global_step % (state.eval_steps) == 0 and state.global_step > 0:
+            self.mllogger.event(
+                "train_loss",
+                value=state.log_history[-1]["loss"] if state.log_history else -1,
+                metadata={
+                    "samples_count": state.log_history[-1]["step"]
+                    * self.global_batch_size
+                    if state.log_history
+                    else -1
+                },
+            )
+            control.should_log = True
+
+        if (
+            state.global_step % (state.eval_steps) == 0
+            and state.global_step > args.eval_delay
```

Comment:
`args.eval_delay` is removed in torch xla pipeline and we can remove it if it is not used in gpu side as well. Other wise we can add a key. 

Feel free to let it as it is if it works well in gpu. I can fix it later.